re.rizin.cutter.plugin.rz-ghidra.yml

```diff
@@ -0,0 +1,30 @@
+id: re.rizin.cutter.plugin.rz-ghidra
+branch: stable
+runtime: re.rizin.cutter
+runtime-version: stable
+appstream-compose: false
+sdk: org.kde.Sdk//5.15-22.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm14
+build-options:
+  prefix: /app/extensions/rz-ghidra
+  prepend-path: /usr/lib/sdk/llvm12/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
```

Comment:
maybe these path need to be update too? If not that probably mean you don't even need the llvm extension

Reply:
removed them they are necessary for the main app but not the plugin. Did not help with the problem though.